src/components/input/bl-input.ts

```diff
@@ -261,22 +268,32 @@ export default class BlInput extends FormControlMixin(LitElement) {
     return this.customInvalidText || this.validationTarget?.validationMessage;
   }
 
+  /**
+   * Sets a custom validity on the form element.
+   * @param message
+   */
+  setCustomValidity(message: string) {
+    this.validationTarget.setCustomValidity(message);
+  }
+
   /**
    * Force to set input as in invalid state.
+   * @deprecated use error attribute instead
    */
   async forceCustomError() {
     await this.updateComplete;
-    this.validationTarget.setCustomValidity(this.customInvalidText || "An error occurred");
+    this.setCustomValidity(this.customInvalidText || "An error occurred");
```

Comment:
we have localized text here, either remove it or wrap with localization

Reply:
It felt like a different context bro

Reply:
it'd be nice to solve it as we caught it, but fine to handle it in a separate pull request.